docs/coordinates/performance.inc.rst

```diff
@@ -243,9 +243,11 @@ for ``time_resolution`` compared to the non-interpolating, default approach.
 
     seps = u.Quantity(seps)
 
-    fig = plt.figure()
-
-    ax1, ax2 = fig.subplots(2, 1, gridspec_kw={'height_ratios': [2, 1]}, sharex=True)
+    fig, (ax1, ax2) = plt.subplots(
+      nrows=2,
+      subplot_kw=dict(sharex=True),
```

Comment:
Why not
```suggestion
      sharex=True,
```
And does the shorter code fit on fewer lines?

Reply:
thanks ! and no, it still doesn't fit a single line